src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,141 @@
+package duke;
+
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.Scanner;
+
+/**
+ * Ui handles all of the text interface functionality of the program
+ *
+ * @author Loh Wen Hao Aaron
+ *
+ */
+public class Ui {
+    private static final String WELCOME_MESSAGE = "Hello, I'm Dobie!\nWhat can I do for you?";
+
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
+    private Duke duke;
+
+    public Ui(Duke duke) {
+        this.duke = duke;
+    }
+
+    public String getWelcomeMessage() {
+        return WELCOME_MESSAGE;
+    }
+
+    /**
+     * Iteration 1:
+     * Creates a label with the specified text and adds it to the dialog container.
+     * @param text String containing text to add
+     * @return a label with the specified text that has word wrap enabled.
```

Comment:
Perhaps you could capitalise the first letter of the sentence after the @return tag like so: 
```suggestion
     * @return A label with the specified text that has word wrap enabled.
```